### PR TITLE
Display error message when trying to create a campaign without devices

### PR DESCRIPTION
**Pull request by piotrcichonmee**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/assets/locales/en/translation.json [hunk 1]
```diff
@@ -12,6 +12,10 @@
             "cancel_tooltip": "When you cancel a campaign, any update operations that are pending or already in progress will still complete. The campaign is canceled only for devices that haven't yet detected the new update.",
             "rename": "Rename campaign"
         },
+        "error_descriptions": {
+            "campaign_without_devices_title": "No devices were found",
+            "campaign_without_devices": "None of the groups selected for the campaign contains any device."
```
Comment (merltron) [inline ota-plus-web/app/assets/locales/en/translation.json]: @piotrcichonmee 
```suggestion
            "campaign_without_devices": "There are no devices in any of the groups that you've selected for this campaign."
```

File: ota-plus-web/app/assets/locales/en/translation.json [hunk 2]
```diff
@@ -12,6 +12,10 @@
             "cancel_tooltip": "When you cancel a campaign, any update operations that are pending or already in progress will still complete. The campaign is canceled only for devices that haven't yet detected the new update.",
             "rename": "Rename campaign"
         },
+        "error_descriptions": {
+            "campaign_without_devices_title": "No devices were found",
```
Comment (merltron) [inline ota-plus-web/app/assets/locales/en/translation.json]:
```suggestion
            "campaign_without_devices_title": "Selected device groups are empty",
```

**Comment by merltron** [comment]
@piotrcichonmee 
Generally, can you please put me (or "tkfu" - Jon Oster) as a reviewer for any PRs that contain messages?
Cheers

File: ota-plus-web/app/assets/locales/en/translation.json (hunk 1 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/assets/locales/en/translation.json]: Done.

File: ota-plus-web/app/assets/locales/en/translation.json (hunk 2 above)
Comment (piotrcichonmee) [reply ota-plus-web/app/assets/locales/en/translation.json]: Done.

**Pull request merged**